apps/public/public.go

```diff
@@ -522,13 +522,44 @@ type interactiveAuthOptions struct {
 	claims, domainHint, loginHint, redirectURI, tenantID string
 	openURL                                              func(url string) error
 	authnScheme                                          AuthenticationScheme
+	successPage                                          []byte
+	errorPage                                            []byte
 }
 
 // AcquireInteractiveOption is implemented by options for AcquireTokenInteractive
 type AcquireInteractiveOption interface {
 	acquireInteractiveOption()
 }
 
+// WithSystemBrowserOptions sets the optional success and error pages.
+// The error page supports two optional html template variables {{.Code}} and {{.Err}},
+// which will be replaced with the corresponding error code, and descriptions.
+func WithSystemBrowserOptions(successPage []byte, errorPage []byte) interface {
```

Comment:
Just FYI, you can use type list syntax for function parameters
```suggestion
func WithSystemBrowserOptions(successPage, errorPage []byte) interface {
```